src/TesApi.Tests/TerraApiStubData.cs

```diff
@@ -68,10 +70,10 @@ public BatchAccountOptions GetBatchAccountOptions()
 
     public string GetWsmSasTokenApiResponseInJson()
     {
-        return @"{
-  ""token"": ""SASTOKENSTUB="",
-  ""url"": ""https://bloburl.foo/container?sas=SASTOKENSTUB=""
-    }";
+        return $@"{{
```

Comment:
Starting with C# 11, you can use multiple $ to indicate the depth of the interpolated brackets, combined with raw string literal, for a nicer format like this:

```
public static string GetWsmSasTokenApiResponseInJson2(string SasToken, string WsmGetSasResponseStorageUrl)
{
    return $$"""
    {
        "token": "{{SasToken}}",
        "url": "{{WsmGetSasResponseStorageUrl}}?sv={{SasToken}}"
    }
    """;
} 
```

Reply:
I tested the above, and it results in this:

```
{
    "token": "sastokenhere",
    "url": "wsmurlhere?sv=sastokenhere"
}
```